Source/Events.Store.MongoDB/Processing/Streams/StreamProcessorState.cs

```diff
@@ -56,5 +57,18 @@ public StreamProcessorState(Guid scopeId, Guid eventProcessorId, Guid sourceStre
         /// Gets or sets a value indicating whether the Stream Processor is failing or not.
         /// </summary>
         public bool IsFailing { get; set; }
+
+        /// <summary>
+        /// Converts the <see cref="StreamProcessorState" /> to the runtime representation of <see cref="Runtime.Events.Processing.Streams.StreamProcessorState"/>.
+        /// </summary>
+        /// <returns>The converted <see cref="Runtime.Events.Processing.Streams.Partitioned.StreamProcessorState" />.</returns>
+        public override IStreamProcessorState ToRuntimeRepresentation() =>
+            new Runtime.Events.Processing.Streams.StreamProcessorState(
```

Comment:
We could have a using statement to shorten this down a bit right?

Reply:
Shortened.